docs/services.html

```diff
@@ -0,0 +1,145 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+
+    <title>Great Idea!</title>
+
+    <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
+    <link rel="stylesheet" href="css/index.css">
+
+    <!--[if lt IE 9]>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+    <![endif]-->
+</head>
+
+<body>
+
+<div class="nav-container">
+
+    <header class="top-nav">
+        <a class="nav-link" href="">Services </a>
```

Comment:
```suggestion
        <a class="nav-link" href="#">Services </a>
```
You should put `href="#"` on anchor tags that are 'dead links'